model/common/src/icon4py/model/common/metrics/metric_fields.py

```diff
@@ -99,7 +99,7 @@ def _compute_ddqz_z_half(
     z_mc: fa.CellKField[wpfloat],
     k: fa.KField[gtx.int32],
     nlev: gtx.int32,
-):  # -> Field[Dims[dims.CellDim, dims.KHalfDim], wpfloat]:
+) -> CellKField[wpfloat]:
```

Comment:
```suggestion
) -> fa.CellKField[wpfloat]:
```
not a big thing but just to be consistent with the rest of the code